Clients/src/presentation/pages/Assessment/index.tsx

```diff
@@ -1,5 +1,72 @@
+import Button from "@mui/material/Button";
+import Stack from "@mui/material/Stack";
+import { Paper, styled, Typography, useTheme } from "@mui/material";
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: '#f9fafc',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  border:"1px solid #EAECF0",
+  boxShadow: "none",
+  paddingRight: "150px",
+  paddingLeft: "25px",
+  paddingTop: "10px",
+  paddingBottom: "10px",
+}));
+
 const Assessment = () => {
-  return <div>Assessment</div>;
+  const theme = useTheme();
+  return (
```

Comment:
Although we're using in-inline style, and also we will in the future, I have some suggestion that does
align with our style of coding and can help to make our code cleaner.

1 - You can keep the `const Item = styled(Paper)(({ theme }) => ...` part, but please do change the hard-coded values
with those provided in the `themes/light.ts` file.
2 - // Define a constant for button styles **to avoid _heavy and complex_ inline styling**:
Like this:
```
const buttonStyles = {
  width: "157px",
  height: "34px",
  marginTop: "30px",
  textTransform: 'none',
  border: "1px solid #175CD3",
  borderRadius: "4px",
  boxShadow: "none",
  gap: "8px",
  paddingTop: "10px",
  paddingBottom: "10px",
  paddingLeft: "16px",
  paddingRight: "16px",
};
```
So its going to look like this:

```
<Button variant='contained' size='medium' sx={buttonStyles}>
            <Typography fontFamily="inter" fontSize="13px" fontWeight="400" lineHeight="20px" textAlign="left">
              Go to assessments
            </Typography>
          </Button>
```

Reply:
Replace hard-coded values with in here also

Reply:
And please do change the `style={}` with `sx={{}}` in anywhere possible